OndaEDFSchemas.jl/Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "OndaEDFSchemas"
 uuid = "9c87d999-769b-4741-85b2-6f554d09e731"
 authors = ["Beacon Biosignals, Inc."]
-version = "0.2.2"
+version = "0.3.0"
```

Comment:
I don't think this is breaking since it's just adding new schemas that use the widened field type rather than changing any existing schemas.

Reply:
Right now this only supports EDF 0.8 (not 0.7 and earlier). This means that the returned value of a number of functions is FilePlanV3 / PlanV3 instead of V2.

Reply:
Oh I see, I somehow missed that this is for OndaEDFSchemas.